source/about/desktop-app-changelog.md

```diff
@@ -8,6 +8,15 @@ This changelog summarizes updates to Mattermost desktop app releases for [Matter
 (release-v5-10)=
 ## Release v5.10
 
+- **v5.10.2, released **
```

Comment:
@amyblais We're missing MM-60233 from the list

Reply:
There is no release note on the PR description https://github.com/mattermost/desktop/pull/3157, but I'll add it to the changelog.

Reply:
Ahh sorry about that, I can add one.

Reply:
Thanks!